docs/explore/duplex-type-and-duplex-field.mdx

```diff
@@ -19,7 +19,7 @@ class User {
   @Field()
   password_hash: string
 
-  @DuplexArrayField({ itemType: String})
+  @DuplexField({ itemType: String })
```

Comment:
If decapi 2 is able to infer type from ts, why is `String` type explicitly used here?